addons/l10n_fr/data/account_fiscal_position_template_data.xml

```diff
@@ -16,8 +16,7 @@
             <field name="name">Domestique - France</field>
             <field name="chart_template_id" ref="l10n_fr_pcg_chart_template"/>
             <field name="auto_apply" eval="True" />
-            <field name="vat_required" eval="True" />
```

Comment:
Hi @kitan191 @william-andre 

I just updated one of my customer v16 instance, and I found that fiscal positions are not working as expected anymore with this change.
Why would you remove the vat_required for domestic taxes ?

Odoo will first search for VAT required fiscal positions (as per https://github.com/odoo/odoo/blob/e4bf46e0f5b171be4114ac0a366cc288972964b3/addons/account/models/partner.py#L237-L243) so it will always return Intra EU-B2B fiscal position if you have none set on customer and VAT is correctly set on company and customer. This is not correct since you will then have all taxes exempted and set to 0%.

The old behavior was not perfect since you end up with EU privé instead of Domestique France if you have no VAT set on customer for instance (for instance if you invoice a NGO without VAT ID) but it was OK from fiscal point of view since it would keep the correct tax on account move and account move lines.

Let me know if you can fix this or if you prefer that I create a new PR ?

Reply:
Good question, either that completely slipped passed me or I'm missing something now.
I don't see anything either in this PR or in the ticket that would justify it @kitan191

Reply:
@william-andre @remi-filament 
It was to apply the "Domestique - France" fiscal position to customers from Monaco, even if no VAT number was provided.
That is the only reason.

If it is not correct, it can be reverted to "VAT required".

Reply:
I no VAT number is provided it will ends up in "EU privé" which is the correct behavior for customers in France without VAT also, so it is fine I suppose !

Reply:
Yes, people from Monaco need a VAT number as well (which actually even starts with `FR`)

Reply:
My bad!

I've made a PR to set it back: https://github.com/odoo/odoo/pull/151518